packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -471,32 +529,38 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   }
 }
 
-.mdc-textfield--multiline {
-  $padding-inset: 4px;
+.mdc-textfield--textarea {
+  $padding-inset: 16px;
   $label-offset-y: $padding-inset + 2;
   $label-offset-x: $padding-inset;
 
   display: flex;
   height: initial;
   transition: none;
+  border: 1px solid $mdc-textfield-divider-on-light;
+  border-radius: 4px;
+
+  @include mdc-theme-dark(".mdc-textfield") {
+    border-color: $mdc-textfield-divider-on-dark;
+  }
 
   &::after {
     content: initial;
   }
 
   // stylelint-disable plugin/selector-bem-pattern
-
   .mdc-textfield__input {
     padding: $padding-inset;
+    padding-top: $padding-inset * 2;
     transition: mdc-textfield-transition(border-color);
-    border: 1px solid $mdc-textfield-divider-on-light;
+    border: 1px solid transparent;
     border-radius: 2px;
 
-    @include mdc-theme-dark(".mdc-textfield") {
-      border-color: $mdc-textfield-divider-on-dark;
+    &:focus {
+      @include mdc-theme-prop(border-color, primary);
     }
 
-    &:focus {
+    &:focus:hover {
```

Comment:
Is this overriding something elsewhere? Otherwise this is repetitive of what's above for &:focus. (Either way, we could put both selectors together...)

Reply:
Turns out it is not. Removing since there isn't a change between just focused or focused while hovering